tests/core/test_views_errors.py

```diff
@@ -40,19 +40,16 @@ def test_backoff_header_is_present_on_error_responses(self):
             self.assertEquals(response.headers['Backoff'], '10')
 
     def test_404_is_valid_formatted_error(self):
-        response = self.app.get('/unknown', status=404)
-        self.assertFormattedError(response, 404, ERRORS.MISSING_RESOURCE, "Not Found",
-                                  "The resource you are looking for could not be found.")
```

Comment:
I don't think the test should be removed from here though ;)

As Ethan suggested, you can add more tests in `tests/test_views_*.py` instead